app/src/controllers/instance/organizations/selectors.js

```diff
@@ -33,3 +36,10 @@ export const querySelector = createAlternativeQueryParametersSelector({
   sortingKey: SORTING_KEY,
   namespace: NAMESPACE,
 });
+
+export const filterQuerySelector = createFilterQuerySelector({
+  defaultPagination: DEFAULT_PAGINATION,
+  defaultSorting: SORTING_ASC,
+  sortingKey: SORTING_KEY,
+  namespace: FILTERED_ORGANIZATIONS,
```

Comment:
Why there is a different namespace here than in above selector?
Why not to use existing `querySelector` specified above?

Reply:
Done.